src/Services/Catalog/CatalogService.Application/UseCases/Author/Commands/Update/UpdateAuthorCommandHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using CatalogService.Domain.Entities;
+using CatalogService.Domain.Interfaces;
+using FluentValidation;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CatalogService.Application.UseCases
+{
+    public class UpdateAuthorCommandHandler(
+        IAuthorRepository authorRepository,
+        IValidator<UpdateAuthorCommand> validator,
+        IMapper mapper) : IRequestHandler<UpdateAuthorCommand>
+    {
+        private readonly IAuthorRepository _authorRepository = authorRepository;
+        private readonly IMapper _mapper = mapper;
+        private readonly IValidator<UpdateAuthorCommand> _validator = validator;
+
+        public async Task Handle(UpdateAuthorCommand request, CancellationToken token)
+        {
+            await _validator.ValidateAsync(request, token);
+
+            var author = _mapper.Map<Author>(request);
```

Comment:
Your update logic is in wrong place, your repository is responsible for db interaction but not for updating each property, you can set db specific logic, for example .Include(x=>x.Prop), but not the main responsibility of your use case handler

fix it in all your update handlers